### ISSUE TITLE
Google Geocode + Request adapter

### ISSUE THREAD

**Issue by boolieman** [nchaulet/node-geocoder#196]
Getting an error using google geocoding with the `request` adapter is not running properly, I am getting a console error saying that I must use a different adapter to support HTTPS properly, `request-promise` module currently does support this.  The version i have is 3.15.0, I can work towards making the update on this.
EDIT: update version from package.json i currently have.

**Comment by nchaulet** [comment]
Could you send me a snippet on how you instantiate your geocoder ?

**Comment by boolieman** [comment]
```var options = {
  provider: 'google',
  httpAdapter: 'request',
  apiKey: 'APIKEY',
  formatter: null
};
 
var geocoder = NodeGeocoder(options);```
This is how I have it setup in my file  I'm trying to access the pool function within the `request` package and was getting caught up on the http/https google error.

**Comment by nchaulet** [comment]
Should be fixed with version 3.15.1 could you confirm ?

**Comment by boolieman** [comment]
Seems to work well for me now! Thanks! Closing this down.
